demo/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://stackoverflow.com/questions/30585289/set-different-minsdkversion-for-testandroid-than-for-main-app
+Ignore the minSdkVersion of UIAutomator (which is 18), otherwise we get a build error, due to this
+project's minSdkVersion being 15.
+-->
+<manifest xmlns:tools="http://schemas.android.com/tools"
+    package="com.braintreepayments.browserswitch.demo">
+
+    <uses-sdk tools:overrideLibrary="android_libs.ub_uiautomator" />
+</manifest>
```

Comment:
An alternative to this file would be to bump `minSdkVersion` to `18`. Maybe that could actually make sense since the min sdk version of the braintree sdk is `21` anyway.

Reply:
I'll bring this up with the team. It might be time for us to bump the min SDK version of this project.

Reply:
We think it's time for a major version bump on this repo, so we can increase `minSdkVersion` to `21`. @calvarez-ov do you want to make that change in this PR?

Reply:
I think it would be better to have a separate PR for that, not tied to adding an instrumentation test.

Could your team please do that?

I think at the same time, you might update the other versions like `compileSdkVersion`, `targetSdkVersion` and `buildToolsVersion` to the latest versions.

Then I can rebase this PR on top of those changes, which will result in a smaller PR.

Thanks!

Reply:
Sure, that sounds like a good plan 👍 I will post here when we have a PR open so that you can rebase.

Reply:
Didn't have news on this in a while, so I opened a separate PR #24 to increase the `minSdkVersion` to `21`.

Reply:
The minSdkVersion bump was merged in master. I've rebased this PR, so this comment outdated now.